components/rhel/rhel-ose/README.rst

```diff
@@ -1,60 +1,69 @@
 Vagrantfile to setup  OpenShift Enterprise
 ==========================================
 
-This Vagrantfile sets up the CDK for development with OpenShift.  This file also sets up private networking that will be used to expose various services to the host.  This Vagrantfile also provisions an instance of `OpenShift Enterprise <https://www.openshift.com/enterprise/whats-new.html>`_.
+This Vagrantfile sets up the CDK for development with OpenShift.
+This file also sets up private networking that will be used to expose various
+services to the host.  This Vagrantfile also provisions an instance of
+`OpenShift Enterprise <https://www.openshift.com/enterprise/whats-new.html>`_.
 
-If you are interested in the process used in the Vagranfile to setup OpenShift, please read the comments at the top of the file.
+If you are interested in the process used in the Vagranfile to setup OpenShift,
+please read the comments at the top of the file.
 
 QuickStart
 ----------
 
-1. Get latest CDK box (Using Developer Subscription).
+1. Get latest CDK box (Using Developer Subscription). You can download the
+   CDK and find further information about CDK `here <http://developers.redhat.com/products/cdk/overview/>`_.
 
 2. Create a directory for the Vagrant Box
 
-  ``$ mkdir directory && cd directory``
+   ``$ mkdir directory && cd directory``
 
 3. Download this Vagrantfile in same directory.
 
-  **Note:** The CDK OpenShift Vagrant Box IP defaults to *10.1.2.2*. If you want a different IP then change the **PUBLIC_ADDRESS** variable in the Vagrantfile.
+   **Note:** The CDK OpenShift Vagrant Box IP defaults to *10.1.2.2*. If you
+   want a different IP then change the **PUBLIC_ADDRESS** variable in the Vagrantfile.
```

Comment:
This is what we have to remove now because after landrush we don't have this `PUBLIC_ADDRESS` variable in Vagrantfile.